src/controllers/mainController.ts

```diff
@@ -398,12 +398,13 @@ export default class MainController implements vscode.Disposable {
     /**
      * Manage connection profiles (create, edit, remove).
      */
-    private onManageProfiles(): Promise<boolean> {
+    private async onManageProfiles(): Promise<void> {
         if (this.canRunCommand()) {
             Telemetry.sendTelemetryEvent('ManageProfiles');
-            return this._connectionMgr.onManageProfiles();
+            await this._connectionMgr.onManageProfiles();
+            return this._objectExplorerProvider.refresh(undefined);
         }
-        return Promise.resolve(false);
+        return Promise.resolve();
     }
```

Comment:
Just remove this line - it's not needed with the method being async and a void return type